source/dem/insertion.cc

```diff
@@ -197,5 +221,131 @@ Insertion<dim>::calculate_insertion_domain_maximum_particle_number(
       inserted_this_step = insertion_information.inserted_this_step;
     }
 }
+
+
+template <int dim>
+void
+Insertion<dim>::find_in_clearing_box_cells(
+  const parallel::distributed::Triangulation<dim> &triangulation)
+{
+  // Clearing the containers
+  in_the_clearing_box.clear();
```

Comment:
I would use "clear_box_cells" and "clear_box_boudary_cells". We do not usually use "the" and "of" for variables, unless it is strictly necessary.